tests/integrations/mcs/tso/keyspace_group_manager_test.go

```diff
@@ -200,19 +200,15 @@ func (suite *tsoKeyspaceGroupManagerTestSuite) TestKeyspacesServedByNonDefaultKe
 						// Make sure every keyspace group is using the right timestamp path
 						// for loading/saving timestamp from/to etcd and the right primary path
 						// for primary election.
-						var (
-							timestampPath string
-							primaryPath   string
-						)
+						var timestampPath string
 						clusterID := strconv.FormatUint(suite.pdLeaderServer.GetClusterID(), 10)
+						rootPath := endpoint.TSOSvcRootPath(suite.pdLeaderServer.GetClusterID())
+						primaryPath := endpoint.KeyspaceGroupPrimaryPath(rootPath, param.keyspaceGroupID)
 						if param.keyspaceGroupID == mcsutils.DefaultKeyspaceGroupID {
```

Comment:
How about adding a function to get the timestamp path for both default and non-default groups?

Reply:
ok, after 6745 merged